api/v1alpha1/packagebundle_test.go

```diff
@@ -11,45 +11,96 @@ import (
 
 func TestPackageBundle_Find(t *testing.T) {
 	var err error
-	sut := api.PackageBundle{
-		Spec: api.PackageBundleSpec{
-			Packages: []api.BundlePackage{
-				{
-					Name: "eks-anywhere-test",
-					Source: api.BundlePackageSource{
-						Registry:   "public.ecr.aws/l0g8r8j6",
-						Repository: "eks-anywhere-test",
-						Versions: []api.SourceVersion{
-							{
-								Name:   "v0.1.0",
-								Digest: "sha256:eaa07ae1c06ffb563fe3c16cdb317f7ac31c8f829d5f1f32442f0e5ab982c3e7",
-							},
-						},
+	givenBundle := func(versions []api.SourceVersion) api.PackageBundle {
+		return api.PackageBundle{
+			Spec: api.PackageBundleSpec{
+				Packages: []api.BundlePackage{
+					{
+						Name: "eks-anywhere-test",
+						Source: api.BundlePackageSource{
+							Registry:   "public.ecr.aws/l0g8r8j6",
+							Repository: "eks-anywhere-test",
+							Versions:   versions},
```

Comment:
```suggestion
							Versions:   versions,
					        },
```
Very much a nit, but....

Reply:
Isn't that supposed to get linted? I did run it with `--fix`

Reply:
I don't think so, strangely enough... There are occasions where you might leave it this way, e.g.:
```go
foo := myType{ foo: "bar",
    ack: "gar"}
```
Notice the assignments start on the first line. I don't love this style, but sometimes it makes sense.